go/worker/common/runtime_host.go

```diff
@@ -268,6 +269,14 @@ func (n *RuntimeHostNode) GetWorkerHostLocked() host.Host {
 	return n.workerHost
 }
 
+// GetCurrentBlock returns the current roothash block from the underlying common node.
+func (n *RuntimeHostNode) GetCurrentBlock() roothash.Block {
+	n.commonNode.CrossNode.Lock()
+	defer n.commonNode.CrossNode.Unlock()
+
+	return *n.commonNode.CurrentBlock
```

Comment:
The way it was used previously, this couldn't happen, I think, but I've now removed this code, since it's no longer needed, as CheckTx is done in the node code.